### ISSUE TITLE
More Earth Vision Models

### ISSUE THREAD

**Issue by Z-Zheng** [microsoft/torchgeo#148]
This is a very nice project to help researchers to reproduce advanced models in remote sensing.
I am an earth vision researcher, also a Ph.D. student from Wuhan University, China. see [home page](http://zhuozheng.top/).
I want to add my research works into torchgeo, such as [FarSeg - CVPR 2020](https://github.com/Z-Zheng/FarSeg), [ChangeStar - ICCV 2021](https://github.com/Z-Zheng/ChangeStar), and more, for a better earth vision community.
I will submit PRs to integrate my research codes into torchgeo.
But before that, I want to know your thoughts.

**Comment by adamjstewart** [comment]
Yes, we would love to have your models in torchgeo! If you open a PR I can review it for you.

**Comment by calebrob6** [comment]
Hey Zhuo, it is great to see you here! I'm a big fan of the work that you all do, and echoing what Adam said, we'd love to have your work in torchgeo. We're trying to build out a production quality library for remote sensing + deep learning research that will make it easy to create benchmark results and reproducible research.

Let us know if we can help at all in the PRs that you open, I know the first time integrating code in a larger library can be challenging and we want to make it as smooth as possible.

**Comment by adamjstewart** [comment]
Closed by #150, feel free to add any other models or datasets you care about!
